### ISSUE TITLE
New Features related to Jet forms

### ISSUE THREAD

**Issue by mandarpatil09** [CrocoBlock/suggestions#2477]
**Date Conditions:** Similar ticket ([Jet Forms #2406](https://github.com/Crocoblock/suggestions/issues/2406))
### Conditional Logic for Date such as 

- Disable Past date / Weekends and Weekdays.
- Disabling specific days
- Date Range functionality.
- Selecting multiple dates.
- Disabling Future dates such as only providing future 7 days or future x days.

### Repeater Limits:

- Club multiple fields in 1 row and calculate in repeater field.
- Set limits to repeater so that not more than 10 or x fields can be allowed. 
- Pass backend value for every field for internal calculation within the form.

**Comment by dasigna** [comment]
yeah. especially the first suggestion to disable past dates would be great for itself!
